adapters/33across/33across.go

```diff
@@ -15,6 +15,7 @@ import (
 
 type TtxAdapter struct {
 	endpoint string
+	Server   config.Server
```

Comment:
You're not going to like this after already completing the work, but I recommend not to add the server variable to the adapter structs. Let's leave it to adapters to capture that information in the future if they'd like. For now, I think we're good with just the updates to the Builder methods.

Reply:
No worries, I've got the muscle memory down for updating the adapters! I'll remove this

Reply:
:joy: